controller/scheduler/fleet.py

```diff
@@ -337,16 +337,16 @@ def state(self, name):
             "deactivating": "down",
         }
         try:
-            # NOTE (bacongobbler): this call to ._get_unit() also acts as a pre-emptive check to
+            # NOTE (bacongobbler): this call to ._get_unit() acts as a pre-emptive check to
             # determine if the job no longer exists (will raise a RuntimeError on 404)
-            unit = self._get_unit(name)
+            self._get_unit(name)
```

Comment:
Looks like calling `_get_unit` will still make the request to the units/ endpoint and raise an error if there was no entry (after retries), but we no longer trust the value it returns for this case.